### PR TITLE
dev/financial#2 Update new payment_processor.title field to be localisable

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
Overview
----------------------------------------
Changes new field to be l10n

Before
----------------------------------------
Field not l10n

After
----------------------------------------
FIeld l10n

Technical Details
----------------------------------------
This field has just been added but @mlutfy pointed out it should be localisable

Comments
----------------------------------------
https://lab.civicrm.org/dev/financial/issues/2

### CODE REVIEW

File: CRM/Financial/DAO/PaymentProcessor.php [hunk 1]
```diff
@@ -185,7 +185,7 @@ public function __construct() {
    */
   public static function getReferenceColumns() {
     if (!isset(Civi::$statics[__CLASS__]['links'])) {
-      Civi::$statics[__CLASS__]['links'] = static::createReferenceColumns(__CLASS__);
+      Civi::$statics[__CLASS__]['links'] = static ::createReferenceColumns(__CLASS__);
```
Comment (seamuslee001) [inline CRM/Financial/DAO/PaymentProcessor.php]: @eileenmcnaughton this seems off

**Comment by seamuslee001** [comment]
Looks good merge on pass

File: CRM/Financial/DAO/PaymentProcessor.php (hunk 1 above)
Comment (eileenmcnaughton) [reply CRM/Financial/DAO/PaymentProcessor.php]: Yeah gencode didn't pick up the changes? I've fixed it manually for now

**Comment by seamuslee001** [comment]
@eileenmcnaughton test failure is related here

**Comment by eileenmcnaughton** [comment]
@seamuslee001 I fixed the immediate issue  & also attempted to follow this https://docs.civicrm.org/dev/en/latest/translation/database/

**Comment by seamuslee001** [comment]
looks good @eileenmcnaughton good we test on a translated database eh?

**Comment by eileenmcnaughton** [comment]
@seamuslee001 for sure - although I made 2 changes - one in response to the test & one because I remembered we'd had to do that before - I don't think the need to add that Schema file would have been picked up

**Comment by eileenmcnaughton** [comment]
@seamuslee001 arrgghh - this was supposed to be worth doing now because that would be easier than later....

**Comment by eileenmcnaughton** [comment]
@mlutfy I tried to convert this but it's not going well

File: CRM/Core/I18n/SchemaStructure_5_13_alpha1.php [hunk 2]
```diff
@@ -0,0 +1,722 @@
...
+
+/**
+ *
+ * @package CRM
+ * @copyright CiviCRM LLC (c) 2004-2019
+ *
+ * Generated from schema_structure.tpl
+ * DO NOT EDIT.  Generated by CRM_Core_CodeGen
+ */
+class CRM_Core_I18n_SchemaStructure {
```
Comment (seamuslee001) [inline CRM/Core/I18n/SchemaStructure_5_13_alpha1.php]: @eileenmcnaughton need to rename this with `_5_13_alpha1` at the end of the class name

**Comment by seamuslee001** [comment]
@eileenmcnaughton the failure here was still expected by me. However now that my PR is merged this should pass

**Comment by seamuslee001** [comment]
Jenkins re test this please

**Comment by eileenmcnaughton** [comment]
@seamuslee001  :-(

**Comment by eileenmcnaughton** [comment]
unrelated fail

**Pull request merged**